auth_mellon.h

```diff
@@ -476,6 +476,7 @@ am_cache_entry_t *am_get_request_session_by_nameid(request_rec *r,
 am_cache_entry_t *am_new_request_session(request_rec *r);
 void am_release_request_session(request_rec *r, am_cache_entry_t *session);
 void am_delete_request_session(request_rec *r, am_cache_entry_t *session);
+void am_invalidate_request_session(request_rec *r, am_cache_entry_t *session);
```

Comment:
You changed this function declaration, but there is still no implementation of this function, what's the point ??